### PR TITLE
Add Trezor Suite

### PR DESCRIPTION
**Pull request by vitkabele**
I'm submitting a [Trezor Suite](https://suite.trezor.io). The software to manage crypto assets on Trezor hardware wallets.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask trezor-suite` is error-free.
- [x] `brew style --fix trezor-suite` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask trezor-suite` worked successfully.
- [x] `brew install --cask trezor-suite` worked successfully.
- [x] `brew uninstall --cask trezor-suite` worked successfully.

### CODE REVIEW

File: Casks/trezor-suite.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  sha256 "23c84ba9bdf952404d7c69d29c8813ef052a4448176af3f08180d45268a8d4cd"
+
+  url "https://suite.trezor.io/web/static/desktop/Trezor-Suite-#{version}-mac.dmg"
+  name "TREZOR Suite"
+  desc "Managing crypto just got safer and easier"
+  homepage "https://suite.trezor.io/"
+
+  livecheck do
+    url :homepage
+    strategy :page_match
```
Comment (SMillerDev) [inline Casks/trezor-suite.rb:12]: Page match is the default, no need to specify it.

```suggestion
```

**Comment by SMillerDev** [comment]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

File: Casks/trezor-suite.rb (hunk 1 above)
Comment (vitkabele) [reply Casks/trezor-suite.rb]: Thank you, fixed.

**Comment by vitkabele** [comment]
> Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

Done. CreateZap did not find anything, so I added all items that I was able to find manually.

**Comment by miccal** [comment]
Thank you @vitkabele.

**Pull request merged**